src/Components/ModalDialog.razor.css

```diff
@@ -0,0 +1,4 @@
+#confirmationModal{
```

Comment:
Perhaps add a space between the selector and the brace.

Reply:
added it in 19c5c7a